src/components/tx/security/redefine/index.tsx

```diff
@@ -11,14 +11,15 @@ import { ErrorBoundary } from '@sentry/react'
 import { REDEFINE_ARTICLE, REDEFINE_SIMULATION_URL } from '@/config/constants'
 import css from 'src/components/tx/security/redefine/styles.module.css'
 import sharedCss from '@/components/tx/security/shared/styles.module.css'
-import RedefineLogoDark from '@/public/images/transactions/redefine-dark-mode.svg'
-import RedefineLogo from '@/public/images/transactions/redefine.svg'
+import RedefineLogoDark from '@/public/images/transactions/redefine-dark-mode.png'
+import RedefineLogo from '@/public/images/transactions/redefine.png'
```

Comment:
Are they imported as URLs in this case?

Reply:
Good question, I think they might be part of the js bundle in this case but will double check. Otherwise we can import them like `<img src="/images/transactions/redefine.png" />`

Reply:
Looks like they are not part of any js chunk when using `next/image`. The URLs are added to the sw.js though I assume for offline support